lib/net.c

```diff
@@ -427,6 +450,37 @@ init_port(struct gatekeeper_if *iface, uint8_t port_id,
 	return 0;
 }
 
+int
+get_ip_type(const char *ip_addr)
```

Comment:
`get_ip_type()` is defined and only used in `lib/net.c`, why not making it static?